From ff323fa4ae85fb293e0f419a5364823b90562216 Mon Sep 17 00:00:00 2001
From: Jim Ramsay <i.am@jimramsay.com>
Date: Tue, 18 May 2010 16:50:20 -0400
Subject: [PATCH 13/14] Bug 594153 (2/3) - Allow setting alarms on any meeting

The "view-time-zone" and "view-categories" actions should not have
been grouped with the "individual" action group but rather the "core"
action group, as they should always be available regardless of whether
or not the element being edited is read-only/owned, as they simply
affect visibility of various widgets.

This fixes a bug introduced in 0597b877c5bf4d21ac4048742ddf6b11e24877ba
where these two actions were accidentally grouped with other actions
that should legitimately be in the "individual" group.
---
 calendar/gui/dialogs/comp-editor.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/calendar/gui/dialogs/comp-editor.c b/calendar/gui/dialogs/comp-editor.c
index 46e340f..7d27474 100644
--- a/calendar/gui/dialogs/comp-editor.c
+++ b/calendar/gui/dialogs/comp-editor.c
@@ -1091,7 +1091,7 @@ static GtkActionEntry individual_entries[] = {
 	  G_CALLBACK (action_attach_cb) }
 };
 
-static GtkToggleActionEntry personal_toggle_entries[] = {
+static GtkToggleActionEntry core_toggle_entries[] = {
 
 	{ "view-categories",
 	  NULL,
@@ -1593,6 +1593,9 @@ comp_editor_init (CompEditor *editor)
 	gtk_action_group_add_actions (
 		action_group, core_entries,
 		G_N_ELEMENTS (core_entries), editor);
+	gtk_action_group_add_toggle_actions (
+		action_group, core_toggle_entries,
+		G_N_ELEMENTS (core_toggle_entries), editor);
 	gtk_ui_manager_insert_action_group (
 		priv->ui_manager, action_group, 0);
 	g_object_unref (action_group);
@@ -1615,9 +1618,6 @@ comp_editor_init (CompEditor *editor)
 	action_group = gtk_action_group_new ("editable");
 	gtk_action_group_set_translation_domain (
 		action_group, GETTEXT_PACKAGE);
-	gtk_action_group_add_toggle_actions (
-		action_group, personal_toggle_entries,
-		G_N_ELEMENTS (personal_toggle_entries), editor);
 	gtk_ui_manager_insert_action_group (
 		priv->ui_manager, action_group, 0);
 	g_object_unref (action_group);
-- 
1.7.1

